next-frontend/src/app/(wca)/regulations/history/official/[version]/page.tsx

```diff
@@ -11,7 +11,7 @@ export default function HistoricalRegulation() {
       <AspectRatio>
         <iframe
           width={"100%"}
-          src={`https://regulations.worldcubeassociation.org/history/official/${params.version}/index.html`}
```

Comment:
I removed the /index.html because that is the default file that cloudfront looks for in the s3 bucket